packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -867,6 +879,24 @@ if (missingLangs.length) {
 const areDistResources = conf.get('static_directory') === 'dist';
 conf.set('are_dist_resources', areDistResources);
 
+if (conf.get('ecosystem_anon_id.keys_file')) {
+  const keysFile = path.resolve(
+    __dirname,
+    '..',
+    'config',
+    conf.get('ecosystem_anon_id.keys_file')
+  );
+  // Alter `ecosystem_anon_id.key` if the keys file exists.
```

Comment:
Should be `ecosystem_anon_id.keys`, will update with any other needed edits after second review.